3_visualize.R

```diff
@@ -93,6 +93,17 @@ p3_targets_list <- list(
                     sites_crs = 4269),
     format = "file"
   ),
+  # with synoptic sampling sites
+  tar_target(
+    p3_nhdv2_attr_syn_sites_png,
+    plot_nhdv2_attr(attr_data = p2_nhdv2_attr_refined %>% select(-hru_segment),
```

Comment:
I see that there are a few locations where we are omitting `hru_segment` from the columns in `p2_nhdv2_attr_refined`, presumably because we don't want to plot the distribution of hru_segment id's. Do you think it makes sense to add `hru_segment` to the vector of strings in `drop_columns` used to build `p2_nhdv2_attr_refined`? Alternatively, maybe we want to retain that attribute elsewhere, just not in the plots in 3_visualize..

Reply:
I think it's better to retain the `hru_segment` column because it's the identifier needed to link with other tables.

Reply:
OK, sounds good!